.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,28 @@
-## Summary
 
-*Short description*
+# Checklist
 
-## TODO
+- [ ] Target milestone
+- [ ] New tests
+- [ ] Updated documentation
+- [ ] External dependency updates
+- [ ] Internal dependency updates (with link to diff)
+- [ ] Screenshots
+- [ ] Update to CHANGELOG.rst
+- [ ] Update to AUTHORS.rst
```

Comment:
Why not "Add yourself it AUTHORS.rst if you don't appear there" ?

Reply:
Just trying to keep things concsise

Reply:
It's so it doesn't become mysterious or exclusive to only the ones who know what the `AUTHORS.rst` file is, rather the only people whom this line is relevant to is actually first time contributors!